_posts/2021-03-09-bezier-fitting.md

```diff
@@ -0,0 +1,196 @@
+---
+layout: post
+title:  "Fitting cubic Bézier curves"
+date:   2021-03-09 16:10:42 -0700
+categories: [curves]
+---
+<!-- I should figure out a cleaner way to do this include, rather than cutting and pasting. Ah well.-->
+<script type="text/x-mathjax-config">
+	MathJax.Hub.Config({
+		tex2jax: {
+			inlineMath: [['$', '$']]
+		}
+	});
+</script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
+
+Cubic beziers are by far the most common curve representation, used both for design and rendering. A fundamental problem is *curve fitting,* or determining the cubic bezier that's closest to some source curve. Applications include simplifying existing paths, efficiently representing the [parallel curve], and rendering other spline representations such as Euler spiral or [hyperbezier]. A specific feature where curve fitting will be used in [Runebender] is deleting a smooth on-curve point. The intent is to merge two beziers into one, which is another way of saying to find a bezier which most closely approximates the curve formed of the two existing bezier segments.
```

Comment:
I would slightly smooth out the transition to the second sentence, maybe something as minor as,

"One of the fundamental problems when working with curves is *curve fitting*," or similar.

Reply:
Runebender link is missing.

 I might also rephrase that sentence so that it does not require familiarity with Runebender, and just say: "One common use-case is in drawing tools, where..."

Reply:
maybe s/most closely/best?, s/formed of/formed from?